cli/core/list.go

```diff
@@ -55,7 +57,7 @@ func runListCommand(cmd *cobra.Command, args []string) {
 
 	logrus.Info("Executing `arduino core list`")
 
-	platforms, err := core.GetPlatforms(inst.Id, listFlags.updatableOnly)
+	platforms, err := core.GetPlatforms(inst.Id, listFlags.updatableOnly, listFlags.all)
```

Comment:
Uhm... I think that this is the wrong path, this is bypassing the RPC interface.
From the `cli` package we must only call gRPC methods, to ensure that IDE (grpc) and CLI features remain coherent.
I think this is a very old technical debt and should be addressed ASAP before it gets worse...